app-editors/gnome-text-editor/gnome-text-editor-47.2-r1.ebuild

```diff
@@ -14,7 +14,7 @@ SLOT="0"
 
 KEYWORDS="~amd64 ~loong ~ppc64 ~riscv ~x86"
 
-IUSE="+editorconfig spell"
```

Comment:
Feel like this merits a TODO in the ebuild or perhaps better to just mask USE=spell for this version, given all of this will have to be added back in the newer versions.